### ISSUE TITLE
.import_ngs_data() HTTPError when filtering for year(s)

### ISSUE THREAD

**Issue by jbf302** [cooperdff/nfl_data_py#26]
HTTPError: HTTP Error 404: Not Found

I receive a 404: not found error when I try to import ngs data from a specific year:

This works just fine and returns the dataframe:
```
ngs_data = nfl.import_ngs_data('passing')

ngs_data
```

When I add a specific year argument, this breaks:
```
ngs_data = nfl.import_ngs_data('passing', [2020])

ngs_data
```

Yes: I realize that  nfl.import_ngs_data('passing') is comprehensive of all seasons that are available and I can filter with that.

I don't know much about these different file formats, but when I look in nflverse-data/releases/nextgen_stats it looks like there are no .parquet files specific to a season (only .qs, .csv.gz, .rds), only the larger ones (sans season) called in the first block:
<img width="1207" alt="Screen Shot 2022-07-26 at 12 12 50 PM" src="https://user-images.githubusercontent.com/36972201/181057281-83eb4525-37b2-4f9b-a881-c94947b846db.png">

So I think something just needs to be [changed around here?](https://github.com/cooperdff/nfl_data_py/blob/04cf18e5afb8c33686704f73d263df6bb8b44895/nfl_data_py/__init__.py#L665)

Thanks for putting together this awesome library!! It's been a huge help as I get ready for the season.

**Comment by cooperdff** [comment]
Thanks for flagging this - the nflverse folks did a little rearranging of their data and it must've made that logic defunct. I'll get it sorted out this weekend.

**Comment by cooperdff** [comment]
Fixed in v0.2.8.
